gr-audio/lib/alsa/alsa_impl.cc

```diff
@@ -162,9 +165,8 @@ bool gri_alsa_pick_acceptable_format(snd_pcm_t* pcm,
         }
     }
 
-    fprintf(stderr,
-            "%s[%s]: failed to find acceptable format",
-            error_msg_tag,
-            snd_pcm_name(pcm));
+    GR_LOG_ERROR(logger,
+                 boost::format("%s[%s]: failed to find acceptable format\n") %
```

Comment:
Hmm… here a newline is actually *added*. Now I'm wondering if all the other lines above that don't are supposed to have them.

Reply:
super confusing, I don't know how that happened.